video/decode/vd_lavc.c

```diff
@@ -156,14 +158,6 @@ static const struct vd_lavc_hwdec mp_vd_lavc_rpi_copy = {
 };
 #endif
 
-#if HAVE_ANDROID
-static const struct vd_lavc_hwdec mp_vd_lavc_mediacodec = {
```

Comment:
Happy to move it back if that makes sense. It seemed better to colocate the two, since that's done with the other backends. But in this case, they share no code.